client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -136,16 +136,18 @@ export class CartItem extends React.Component {
 
 	render() {
 		let name = this.getProductName();
-		if ( this.props.cartItem.bill_period && this.props.cartItem.bill_period !== -1 ) {
+		if ( this.props.cartItem.bill_period && parseInt( this.props.cartItem.bill_period ) !== -1 ) {
```

Comment:
Probably could use lodash's `get` to simplify this? `if ( parseInt( get( cartItem, 'bill_period' ) ) !== -1 ) { }`... Not sure if better 😅